Core/include/Acts/TrackFitting/GlobalChiSquareFitter.hpp

```diff
@@ -770,6 +787,8 @@ class Gx2Fitter {
 
     ACTS_VERBOSE("final covariance:\n" << fullCovariancePredicted);
 
+    trackContainer.template addColumn<size_t>("Gx2fnUpdateColumn");
```

Comment:
Following the pattern in the other fitters, you'd check here if the column is defined and only fill it if it is fact defined. This way, the client can designed whether they want the info attached or not. What do you think?